src/Symfony/Component/Translation/Bridge/Lokalise/LokaliseProvider.php

```diff
@@ -149,6 +156,7 @@ private function exportFiles(array $locales, array $domains): array
                 'filter_langs' => array_values($locales),
                 'filter_filenames' => array_map([$this, 'getLokaliseFilenameFromDomain'], $domains),
                 'export_empty_as' => 'skip',
+                'placeholder_format' => $this->intlIcuEnabled ? 'icu' : 'symfony',
```

Comment:
A different parameter is required to ensure that the plurals are also in the expected ICU format. The param is called `plural_format` which should contain the same value as  placeholder_format. See https://developers.lokalise.com/reference/download-files for all params. It would be great if that can be included. Without this option users might get invalid ICU translation files